spec/core_ext/string/formats_spec.rb

```diff
@@ -21,6 +21,16 @@
     expect("foo").not_to be_domain_name
   end
 
+  it '#hostname?' do
```

Comment:
Can you add a test for an IPv4 and IPv6 address?

Currently the following is `true` and I think it should return `false`.
```ruby
"1.1.1.1".hostname?
=> true
```

Reply:
This value is also an acceptable value for hostname, I'm based on the RFC definition:
https://tools.ietf.org/html/rfc1123#section-2.1

Reply:
I'm sorry, my bad, there is a stretch about this case:
> However, a valid host name can never
           have the dotted-decimal form #.#.#.#, since at least the
           highest-level component label will be alphabetic.

Then I wil adapt the rgx

Reply:
@bdunne it's done! :smile: